### DISCUSSION TITLE
Does Lua have access to Feed To Listbox feature

### DISCUSSION THREAD

**Discussion by advice2020** [doublecmd/doublecmd#1813] [General]
I really like how DC can send generated lists of items to a new tab within file pane.
This feature can be seen in the Search/Find Dialog when results are generated and can be sent to tab using the "Feed To Listbox" button.

Although I have feature request ideas on how this could be used in DC internal tools, I do not know exactly how I would want to use this yet, but I was wondering if Lua scripts in any way had access to anything similar in DC, that when results are generated from some kind of tool, that they can be sent to a new tab in the same way?

Thank You

**Comment by j2969719** [comment]
https://doublecmd.github.io/doc/en/cmds.html#cm_LoadList

**Comment by advice2020** [comment]
@j2969719
When I was making this post and I was using the Find/Search as an example I knew I came across another way this was accessed in DC just could not remember where.  I must have come across that command in the past, unless there is another situation where DC uses this “Feed To Listbox” action as well.  
Thank You

-----------------------------------------------------------------------------
Anyone
Not important, but does anyone know of any other instances where this “Feed To Listbox” action is used in DC?
Just curious what else I could have been thinking of if it was not this internal command mentioned.

Anyways, okay so this internal command allows this action to be used for opening up text files that contain path/filenames.  So this is how Lua would get access to this “Feed To Listbox” action?  So you would have the Lua script save to something like maybe a temporary text file and then use this internal DC command to access this text file which would load it into file pane in a new tab.

Like I said I do not have any specific ideas yet for this but I really like this feature in DC and feel it could be used for other things as well.
Thank you again
